File: src/main/java/com/eric/okpo_java/lab1/Task4.java
package com.eric.okpo_java.lab1;

import java.util.Scanner;

public class Task4 {

    public static void main(String[] args) {
        double radA, sum;
        var input = new Scanner(System.in);
        radA = Math.toRadians(input.nextDouble());
        sum = Math.pow(Math.sin(radA), 2) + Math.pow(Math.cos(radA), 2);

        System.out.println(sum);
    }
}

File: src/main/java/com/eric/okpo_java/lab4/Task4.kt
package com.eric.okpo_java.lab4

class Task4 {

    fun main() {
        var sum = 0.0
        var x: Double

        while (sum < 100) {
            print("Введите x: ")
            x = readln().toDouble()

            if (x >= 0) sum += x
            else println("Вы ввели не положительное число")
        }

        println("Sum: $sum")
    }
}